targets/csharp-unityv2/Settings/Admin/PlayFabSettings.cs

```diff
@@ -0,0 +1,19 @@
+#if ENABLE_PLAYFABADMIN_API
+using System;
+using UnityEngine;
+using System.Collections;
+using PlayFab.Internal;
+
+namespace PlayFab
+{
+    public static partial class PlayFabSettings
+    {
+        [Obsolete("Use DeveloperSecretKey, this will go away on release of the beta.")]
```

Comment:
Wait... what?  Why would you even add a new thing that is obsolete in it's first release?